src/apiClient.js

```diff
@@ -2,8 +2,6 @@ import Constants from './constants';
 import Types from './types';
 import { BatchUploader } from './batchUploader';
 
-var Messages = Constants.Messages;
```

Comment:
If you're removing this instance of `Messages`, are you removing any related messages from `constants.ts`?

Reply:
yes. I removed the eventsv3 flag there